packages/core/admin/admin/src/translations/en.json

```diff
@@ -182,11 +182,11 @@
   "Settings.permissions.auditLogs.details": "Log Details",
   "Settings.permissions.auditLogs.payload": "Payload",
   "Settings.permissions.auditLogs.listview.header.subtitle": "Logs of all the activities that happened in your environment",
-  "Settings.permissions.auditLogs.entry.create": "Create entry ({model})",
-  "Settings.permissions.auditLogs.entry.update": "Update entry ({model})",
-  "Settings.permissions.auditLogs.entry.delete": "Delete entry ({model})",
-  "Settings.permissions.auditLogs.entry.publish": "Publish entry ({model})",
-  "Settings.permissions.auditLogs.entry.unpublish": "Unpublish entry ({model})",
+  "Settings.permissions.auditLogs.entry.create": "Create entry {model}",
+  "Settings.permissions.auditLogs.entry.update": "Update entry {model}",
+  "Settings.permissions.auditLogs.entry.delete": "Delete entry {model}",
+  "Settings.permissions.auditLogs.entry.publish": "Publish entry {model}",
+  "Settings.permissions.auditLogs.entry.unpublish": "Unpublish entry {model}",
```

Comment:
One tiny drawback is that we have a useless space at the end of the strings without a model

Reply:
I agree what I have now really isn't ideal but I just wanted to keep moving forward. I'll see what I can do.